README.md

```diff
@@ -2,18 +2,11 @@
 
 **hipBLAS** is a **BLAS** marshalling library, with multiple supported backends.  It sits between the application and a 'worker' BLAS library, marshalling inputs into the backend library and marshalling results back to the application.  **hipBLAS** exports an interface that does not require the client to change, regardless of the chosen backend.  Currently, **hipBLAS** supports **rocBLAS** and **cuBLAS** as backends.
 
-
-| Acronym  | Expansion                                                   |
-| :------- | :---------------------------------------------------------- |
-| **BLAS** | **B**asic **L**inear **A**lgebra **S**ubprograms            |
-| **ROCm** | **R**adeon **O**pen E**C**osyste**m**                       |
-| **HIP**  | **H**eterogeneous-Compute **I**nterface for **P**ortability |
-
 ## Documentation
 
 For a detailed description of the **hipBLAS** library, its implemented routines, the installation process and user guide, see the [**hipBLAS** Documentation](https://hipblas.readthedocs.io/en/latest/).
 
-hipBLAS requires either **rocBLAS** + **rocSOLVER** or **cuBLAS** APIs for BLAS implementation. For more information dependent **roc*** libraries see [rocBLAS documentation](https://rocblas.readthedocs.io/en/latest/), and [rocSolver documentation](https://rocsolver.readthedocs.io/en/latest/).
+hipBLAS requires either **rocBLAS** + **rocSOLVER** or **cuBLAS** APIs for BLAS implementation. For more information dependent **roc*** libraries see [rocBLAS and rocSOLVER documentation](https://docs.amd.com/category/libraries).
```

Comment:
I would wait to change these to docs.amd.com until the library documentation on the new site is a little more complete. https://docs.amd.com/bundle/RocBLAS_documentation/page/functions.html#rocblas-int

Reply:
Changed back for now.